### DISCUSSION TITLE
Using onPagePrerender

### DISCUSSION THREAD

**Discussion by kochrt** [vikejs/vike#1269] [Discussion]
https://vike.dev/prerender-programmatic#onpageprerender

What are the risks with using onPagePrerender? Or what should I be concerned about?

I would like to direct the output of prerendering somewhere specific, not necessarily `dist/client`. I am calling `prerender` programmatically.

**Comment by brillout** [comment]
AFAIK only one user uses that API, so this API should be considerer highly experimental.

> I would like to direct the output of prerendering somewhere specific, not necessarily `dist/client`. I am calling `prerender` programmatically.

Can you use an additional separate post build step instead?
